src/platform/qt/MemorySearch.h

```diff
@@ -19,7 +19,8 @@ class MemorySearch : public QWidget {
 Q_OBJECT
 
 public:
-	static constexpr size_t LIMIT = 10000;
+	static constexpr size_t LIMIT = 0x48000; // WRAM size 256KB +32KB
```

Comment:
Why are you increasing this to this? "Every byte in RAM matches exactly once" is pretty arbitrary.